include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -15,6 +15,7 @@
 #endif
 
 #include <string>
+#include <vector>
```

Comment:
Right, indeed, `printValue_impl` was supposed to do that and it fails: the second "overload" is meant to be the case where `&v[0]` doesn't work. But indeed, that code is invalid because both "overloads" have the same signature, and the second doesn't get disabled if the first is valid.

We should "just" fix that. Do you want to try?